src/actions/apiDataFail.test.ts

```diff
@@ -0,0 +1,40 @@
+import { apiDataFail } from './apiDataFail';
+
```

Comment:
pls add a `describe` around the test that describes the module these tests are for. Or add the module name to your test description (if you have only one test): 'apiDataFail action creator should return the ApiDataFail action object without a response'

Reply:
pls do this for all the tests